### PR TITLE
improve events, add nonce to the bond methods

### PR DESCRIPTION
**Pull request by ddulesov**
Note!  day duration  default value is 60 sec. This is  intentionally set for test and development purpose.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM debian:stable-slim
+EXPOSE 9944 9933 30300
+COPY evercity-node .
+VOLUME /var/run/evercity
+CMD [ "/evercity-node", "--dev", "--rpc-external", "--ws-external", "--ws-port", "9944",  "--rpc-port", "9933", "--port", "30300", "--base-path", "/var/run/evercity" ]
```
[the remark this answers is not in the document]
Comment (ddulesov) [reply docker/Dockerfile:12]: I've added some comments to the Dockerfile

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM debian:stable-slim
+EXPOSE 9944 9933 30300
```
[the remark this answers is not in the document]
Comment (ddulesov) [reply docker/Dockerfile:2]: I suppose that using docker bridge driver features as  port mapping is the best options.
that is    --publish hostport:containerport 
All known to me docker images use  static ports.

File: pallets/evercity/src/lib.rs [hunk 3]
```diff
@@ -1055,7 +1124,7 @@ decl_module! {
         /// Calculates bond coupon yield
         #[weight = 25_000]
         fn bond_accrue_coupon_yield(origin, bond: BondId) -> DispatchResult {
-            let _caller = ensure_signed(origin)?;
+            let _ = ensure_signed(origin)?;
```
[the remark this answers is not in the document]
Comment (ddulesov) [reply pallets/evercity/src/lib.rs:1141]: This is an auxiliary function that can be called by any who want to spend his Gas to  get  interest rate and accrued yield.
It does nothing but  calculate bond yields. The caller won't get any rewards. 
Master can use this function to plot a grath of coupon yield and interest rate.
Issuer/Manager can use it for calculation his debt

File: pallets/evercity/src/lib.rs [hunk 4]
```diff
@@ -1266,11 +1334,10 @@ decl_module! {
                            };
 
                            to_packages.push(
-                                BondUnitPackageOf::<T>{
+                                BondUnitPackage{
                                      bond_units,
                                      acquisition,
                                      coupon_yield,
```
[the remark this answers is not in the document]
Comment (ddulesov) [reply pallets/evercity/src/lib.rs]: It is not an issue. Exact time as a timestamp can be easily obtaned by  from acquisition value by this way
timestamp = Bond.active_start_date +  BondUnitPackage.acquisition.

**Review by BoogerWooger — APPROVED** [review]

**Pull request merged**